genesis/process/disabled/disabledBalanceComputationHandler.go

```diff
@@ -0,0 +1,40 @@
+package disabled
+
+import "math/big"
+
+// BalanceComputationHandler -
+type BalanceComputationHandler struct {
+}
+
+// Init -
```

Comment:
Since these are not mocks. Might update the comments to something like "xxx does nothing" or "yyy returns nil"

Reply:
done